images/diagnostics/Dockerfile

```diff
@@ -0,0 +1,9 @@
+#
+# OpenShift diagnostics image
+# Used by network diagnostics (oadm diagnostics NetworkCheck)
+#
+# The standard name for this image is openshift/diagnostics-deployer
+
+FROM docker.io/busybox
```

Comment:
origin-base doesn't have anything OpenShift-specific, it's just centos7 plus a few packages. As with sdodson's comment about using rhel7 base images in OSE, part of the point is that by having a common base, we make it likely that the base will already have been pulled